app.py

```diff
@@ -160,10 +164,13 @@ def get_data(ticker, range=0.05, maxSize=32, minVolumePerc=0.01):
     vol_grp_ask = vol_grp_ask[
         ((vol_grp_ask[TBL_VOLUME] >= minVolume) & (vol_grp_ask['count'] >= 2.0) & (vol_grp_ask['count'] < 70.0))]
     vol_grp_ask['unique'] = vol_grp_ask.index.get_level_values(TBL_VOLUME)
-    vol_grp_ask['unique'] = vol_grp_ask['unique'].apply(round_sig, args=(3,))
+    vol_grp_ask['unique'] = vol_grp_ask['unique'].apply(round_sig, args=(1,2))
```

Comment:
'Unique' contains the exact order size, wich needs 3 significant because of distuingtion beetween 1.5 wich is human behaviour and 2.536 wich typically is a bot

So I disagree with this line change

Reply:
I changed this because for example, with args=(3,) it says 97.2 ETH: https://i.imgur.com/bwfCTlv.png but with args=(1,2) it correctly says 97.25 ETH...

Reply:
But using my overwrite parameter is the wrong way cause for btc USD with orders like 0.0987264 your line would round it to 0.09 mine to 0.0987

The minimum parameter would be the correct one

Reply:
Ah, yes, args=(3,0,2) is fine, args=(3,) like it was before wasn't. Will update.